operations/src/test/resources/com/nextdoor/bender/operations/gelf/prefixed_input.json

```diff
@@ -1 +1 @@
-{"version":"1.1","host":"localhost","short_message":"foo","full_message":"foo bar","timestamp":1234567.89,"level":1,"facility":"local0","line":12,"file":"foo.py","extra_str":"extra field","extra_num":123,"extra_bool":true}
```

Comment:
I don't think this should have been changed because it does test to ensure that if `timestamp` is provided it is not overwritten.